src/tests/functional/shared_test_classes/include/shared_test_classes/base/utils/ranges.hpp

```diff
@@ -34,7 +36,7 @@ struct InputGenerateData {
             : start_from(_start_from), range(_range), resolution(_resolution), seed(_seed) {}
 };
 
-static std::map<ov::NodeTypeInfo, std::vector<std::vector<InputGenerateData>>> inputRanges = {
+static std::unordered_map<ov::NodeTypeInfo, std::vector<std::vector<InputGenerateData>>> inputRanges = {
```

Comment:
Honestly, I do not see any proves to replace `map` by `unordered_map`. Why do you make the changes?

Reply:
in general using <map> means that one cares about the order.
Here we actually do not care as far as I understand.
Just to avoid any confusion.

Reply:
@EgorDuplensky 
I know the difference between 2 types :)
I don't think that it's crucial change. It is the main idea